src/connector/src/parser/protobuf/recursive.rs

```diff
@@ -0,0 +1,157 @@
+#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
This file seems unrelated. Is there something wrong with the rebase/merge?

Reply:
Wired, let me remove it.